app/tsconfig.json

```diff
@@ -11,13 +11,13 @@
    },
    "files": [
 	  "./src/ts/app.ts",
-    "./src/ts/upload.ts",
-    "./src/ts/play.ts",
+      "./src/ts/upload.ts",
```

Comment:
Just deleting all the whitespace and then realigning it might be the easiest thing. I don't think it is that pressing though.